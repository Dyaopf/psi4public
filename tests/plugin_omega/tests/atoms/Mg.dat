molecule {
0 1
Mg
symmetry c1
}

plugin_load("../../plugin_omega.so")

set globals {
  scf_type direct
  basis cc-pvdz
  ri_basis_scf cc-pvdz-ri
  reference uks
  dft_functional wB97
  dft_order_spherical 31 
  dft_n_radial 50
  omega_procedure ip
  d_converge 5
  e_converge 7
  #debug 3
}


energy('scf')
plugin("../../plugin_omega.so")
