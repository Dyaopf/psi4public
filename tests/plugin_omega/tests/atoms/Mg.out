    -----------------------------------------------------------------------
          PSI4: An Open-Source Ab Initio Electronic Structure Package
                              PSI 4.0.0-alpha Driver
    T. D. Crawford, C. D. Sherrill, E. F. Valeev, J. T. Fermann, R. A. King,
    M. L. Leininger, S. T. Brown, C. L. Janssen, E. T. Seidl, J. P. Kenny,
    and W. D. Allen, J. Comput. Chem. 28, 1610-1616 (2007)

                         Additional Contributions by
    Francesco Evangelista, Andrew Simmonett, Justin Turney, Jeremiah Wilke
    -----------------------------------------------------------------------


    Using LocalCommunicator (Number of processes = 1)

    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
molecule {
0 1
Mg
symmetry c1
}

plugin_load("../../plugin_omega.so")

set globals {
  scf_type direct
  basis cc-pvdz
  ri_basis_scf cc-pvdz-ri
  reference uks
  dft_functional wB97
  dft_order_spherical 31 
  dft_n_radial 50
  omega_procedure ip
  d_converge 5
  e_converge 7
  #debug 3
}


energy('scf')
plugin("../../plugin_omega.so")
--------------------------------------------------------------------------
../../plugin_omega.so loaded.

*** tstart() called on ender
*** at Tue Aug  9 12:34:47 2011


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonnett
                              UKS Reference
                        4 Threads,    256 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z       
    ------------   -----------------  -----------------  -----------------
          MG          0.000000000000     0.000000000000     0.000000000000

    Number of unique atoms: 1

    Atoms equivalency:
       unique atom 0: 0 

  Running in c1 symmetry.

  Nuclear repulsion =    0.000000000000000

  Charge       = 0
  Multiplicity = 1
  Electrons    = 12
  Nalpha       = 6
  Nbeta        = 6

  ==> Algorithm <==

  SCF Algorithm Type is DIRECT.
  DIIS enabled.
  MOM disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-07
  Density threshold  = 1.00e-05
  Integral threshold = 0.00e+00

  ==> Primary Basis: CC-PVDZ <==

  Basis Set
    Number of shells: 8
    Number of basis function: 18
    Number of Cartesian functions: 19
    Spherical Harmonics?: true
    Max angular momentum: 2

  Selected Functional is wB97.

  ==> Pre-Iterations <==

  Minimum eigenvalue in the overlap matrix is 1.3235326892E-01.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  Initial UKS energy:  -141.13343867758513

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         18      18       6       6       6       0
   -------------------------------------------------------
    Total      18      18       6       6       6       0
   -------------------------------------------------------

  ==> Integral Setup <==

  Building Direct Integral Objects...

  ==> Iterations <==

                        Total Energy        Delta E     Density RMS

   @UKS iter   1:  -199.66118463457630   -5.85277e+01   0.00000e+00  
   @UKS iter   2:  -200.03353918151643   -3.72355e-01   1.09429e-01 DIIS
   @UKS iter   3:  -200.04962481112798   -1.60856e-02   2.35406e-02 DIIS
   @UKS iter   4:  -200.05139550496591   -1.77069e-03   8.58674e-03 DIIS
   @UKS iter   5:  -200.05140206196643   -6.55700e-06   5.56752e-04 DIIS
   @UKS iter   6:  -200.05140213688168   -7.49152e-08   4.02230e-05 DIIS

  ==> Post-Iterations <==

	Orbital Energies (a.u.)
	-----------------------

	Alpha Occupied:                                                       

	   1A    -46.489456     2A     -3.161305     3A     -1.931960  
	   4A     -1.931960     5A     -1.931960     6A     -0.270279  

	Alpha Virtual:                                                        

	   7A      0.033551     8A      0.033551     9A      0.033551  
	  10A      0.171782    11A      0.260500    12A      0.260500  
	  13A      0.260500    14A      0.427111    15A      0.427111  
	  16A      0.427111    17A      0.427111    18A      0.427111  

	Beta Occupied:                                                        

	   1A    -46.489456     2A     -3.161305     3A     -1.931960  
	   4A     -1.931960     5A     -1.931960     6A     -0.270279  

	Beta Virtual:                                                         

	   7A      0.033551     8A      0.033551     9A      0.033551  
	  10A      0.171782    11A      0.260500    12A      0.260500  
	  13A      0.260500    14A      0.427111    15A      0.427111  
	  16A      0.427111    17A      0.427111    18A      0.427111  

	Final Occupation by Irrep:
	          A 
	DOCC [     6 ]
	SOCC [     0 ]

  Energy converged.

  @UKS Final Energy:  -200.05140213688168

  ==> Properties <==

 OEPROP: One-electron properties/analyses.
  by Rob Parrish and Justin Turney.
  built on LIBMINTS.

 Dipole Moment: (a.u.)
     X:    -0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000

 Dipole Moment: (Debye)
     X:    -0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000


  @Spin Contamination Metric: -0.00000
  @S^2 Expected:               0.00000
  @S^2 Observed:              -0.00000

  Saving occupied orbitals to File 100.

*** tstop() called on ender at Tue Aug  9 12:34:56 2011
	user time   =       8.73 seconds =       0.15 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =          9 seconds =       0.15 minutes
Calling plugin ../../plugin_omega.so.

*** tstart() called on ender
*** at Tue Aug  9 12:34:56 2011


         ---------------------------------------------------------
                          Optimized-Omega RC-KS
                            IP-Only Algorithm                
                               Rob Parrish
                        4 Threads,    256 MiB Core
         ---------------------------------------------------------

 ==> Functional <==

  wB97

 ==> Geometry <==

    Molecular point group: c1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z       
    ------------   -----------------  -----------------  -----------------
          MG          0.000000000000     0.000000000000     0.000000000000

    Number of unique atoms: 1

    Atoms equivalency:
       unique atom 0: 0 

  Nuclear repulsion =    0.000000000000000

  Nalpha = 6
  Nbeta  = 6

  ==> Primary Basis <==

  Basis Set
    Number of shells: 8
    Number of basis function: 18
    Number of Cartesian functions: 19
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Primary Basis <==

  Basis Set
    Number of shells: 24
    Number of basis function: 76
    Number of Cartesian functions: 90
    Spherical Harmonics?: true
    Max angular momentum: 3

  ==> Omega Guess: HOMO Size <== 

   HOMO <R>:          3.44687E+00
   Initial Omega^-1:  3.44687E+00
   Initial Omega:     2.90118E-01

  Canonical Orthogonalization: 18 of 18 functions retained, 0 projected

  ==> Burn-In Procedure <==

  *** Step    1: Burn-In Step ***

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.06965310855526 -2.00070E+02  1.25331E-03 
      2  -200.06968227728663 -2.91687E-05  4.79343E-04 DIIS
      3  -200.06968458375596 -2.30647E-06  1.87401E-04 DIIS
      4  -200.06968479133181 -2.07576E-07  4.65610E-07 DIIS
      5  -200.06968479133008  1.73372E-12  3.72257E-07 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.78265517770166 -1.99783E+02  1.57070E-02 
      2  -199.78925034370835 -6.59517E-03  6.04901E-03 DIIS
      3  -199.78987263565122 -6.22292E-04  2.86437E-03 DIIS
      4  -199.78994089605692 -6.82604E-05  5.50678E-05 DIIS
      5  -199.78994093151306 -3.54561E-08  2.71308E-06 DIIS

  ==> Omega Optimization Procedure <==

  *** Step    2: Bracketing Step ***

    Old Omega:     2.90118472E-01
    Old kIP:       2.62927819E-01
    Old IP:        2.79743860E-01
    Old Delta IP: -1.68160409E-02
    Multipler:     2.00000000E+00
    New Omega:     5.80236944E-01

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.06441359882862  5.27119E-03  1.61531E-03 
      2  -200.06446132114928 -4.77223E-05  6.04407E-04 DIIS
      3  -200.06446491533595 -3.59419E-06  2.17216E-04 DIIS
      4  -200.06446518354088 -2.68205E-07  1.31130E-06 DIIS
      5  -200.06446518351095  2.99281E-11  2.32939E-08 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.78219490221173  7.74603E-03  8.84671E-04 
      2  -199.78223414082385 -3.92386E-05  2.45807E-04 DIIS
      3  -199.78223511262243 -9.71799E-07  8.23074E-05 DIIS
      4  -199.78223518497637 -7.23539E-08  1.78121E-05 DIIS
      5  -199.78223518693275 -1.95638E-09  1.16535E-09 DIIS

  *** Step Results ***

    Omega:     5.80236944E-01
    kIP:       2.73621216E-01
    IP:        2.82229997E-01
    Delta IP: -8.60878038E-03

  *** Step    3: Bracketing Step ***

    Old Omega:     5.80236944E-01
    Old kIP:       2.73621216E-01
    Old IP:        2.82229997E-01
    Old Delta IP: -8.60878038E-03
    Multipler:     2.00000000E+00
    New Omega:     1.16047389E+00

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.11973681562475 -5.52716E-02  1.68952E-03 
      2  -200.12004337452694 -3.06559E-04  3.87132E-04 DIIS
      3  -200.12004565750408 -2.28298E-06  1.45554E-04 DIIS
      4  -200.12004583811563 -1.80612E-07  3.12627E-05 DIIS
      5  -200.12004584427555 -6.15992E-09  6.73685E-07 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.83827007020199 -5.60349E-02  1.50984E-03 
      2  -199.83853347452123 -2.63404E-04  2.58585E-04 DIIS
      3  -199.83853554810898 -2.07359E-06  9.54624E-05 DIIS
      4  -199.83853566422007 -1.16111E-07  2.78608E-05 DIIS
      5  -199.83853567066777 -6.44769E-09  5.31153E-07 DIIS

  *** Step Results ***

    Omega:     1.16047389E+00
    kIP:       2.73982628E-01
    IP:        2.81510174E-01
    Delta IP: -7.52754586E-03

  *** Step    4: Bracketing Step ***

    Old Omega:     1.16047389E+00
    Old kIP:       2.73982628E-01
    Old IP:        2.81510174E-01
    Old Delta IP: -7.52754586E-03
    Multipler:     2.00000000E+00
    New Omega:     2.32094778E+00

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.11845758134311  1.58826E-03  2.95490E-03 
      2  -200.11903940199511 -5.81821E-04  1.22899E-04 DIIS
      3  -200.11904038045026 -9.78455E-07  1.80191E-05 DIIS
      4  -200.11904038963610 -9.18584E-09  5.66563E-07 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.83607733951143  2.45833E-03  2.31030E-03 
      2  -199.83666496625744 -5.87627E-04  1.42028E-04 DIIS
      3  -199.83666829434600 -3.32809E-06  6.85183E-06 DIIS
      4  -199.83666831262866 -1.82827E-08  5.99363E-07 DIIS

  *** Step Results ***

    Omega:     2.32094778E+00
    kIP:       2.75020201E-01
    IP:        2.82372077E-01
    Delta IP: -7.35187555E-03

  *** Step    5: Bracketing Step ***

    Old Omega:     2.32094778E+00
    Old kIP:       2.75020201E-01
    Old IP:        2.82372077E-01
    Old Delta IP: -7.35187555E-03
    Multipler:     2.00000000E+00
    New Omega:     4.64189555E+00

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.12230112123351 -3.26073E-03  9.21062E-04 
      2  -200.12232778985316 -2.66686E-05  1.81096E-04 DIIS
      3  -200.12232816948955 -3.79636E-07  7.38539E-05 DIIS
      4  -200.12232820039154 -3.09020E-08  4.59930E-06 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.83911842009135 -2.45011E-03  5.68739E-04 
      2  -199.83914947996638 -3.10599E-05  9.33447E-05 DIIS
      3  -199.83914984494953 -3.64983E-07  2.98190E-05 DIIS
      4  -199.83914985739494 -1.24454E-08  1.05952E-05 DIIS
      5  -199.83914985793336 -5.38421E-10  1.37878E-06 DIIS

  *** Step Results ***

    Omega:     4.64189555E+00
    kIP:       2.75716799E-01
    IP:        2.83178342E-01
    Delta IP: -7.46154313E-03

  *** Step    6: Bracketing Step ***

    Old Omega:     4.64189555E+00
    Old kIP:       2.75716799E-01
    Old IP:        2.83178342E-01
    Old Delta IP: -7.46154313E-03
    Multipler:     2.00000000E+00
    New Omega:     9.28379110E+00

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.16197166884450 -3.96435E-02  2.63114E-04 
      2  -200.16197809921681 -6.43037E-06  7.64400E-05 DIIS
      3  -200.16197818063415 -8.14173E-08  3.05991E-05 DIIS
      4  -200.16197818647424 -5.84009E-09  5.77347E-06 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.87843160002282 -3.92817E-02  1.89992E-04 
      2  -199.87844055024345 -8.95022E-06  3.59350E-05 DIIS
      3  -199.87844059847484 -4.82314E-08  1.46055E-05 DIIS
      4  -199.87844060134955 -2.87471E-09  2.57328E-06 DIIS

  *** Step Results ***

    Omega:     9.28379110E+00
    kIP:       2.75999822E-01
    IP:        2.83537585E-01
    Delta IP: -7.53776344E-03

  *** Step    7: Bracketing Step ***

    Old Omega:     9.28379110E+00
    Old kIP:       2.75999822E-01
    Old IP:        2.83537585E-01
    Old Delta IP: -7.53776344E-03
    Multipler:     2.00000000E+00
    New Omega:     1.85675822E+01

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.11394706166845  4.80311E-02  6.27086E-05 
      2  -200.11394750285783 -4.41189E-07  1.80563E-05 DIIS
      3  -200.11394750743216 -4.57433E-09  6.03883E-06 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.83030008527089  4.81405E-02  4.08522E-05 
      2  -199.83030083561871 -7.50348E-07  8.22066E-06 DIIS
      3  -199.83030083594485 -3.26139E-10  3.28625E-06 DIIS

  *** Step Results ***

    Omega:     1.85675822E+01
    kIP:       2.76092803E-01
    IP:        2.83646671E-01
    Delta IP: -7.55386890E-03

  *** Step    8: Bracketing Step ***

    Old Omega:     1.85675822E+01
    Old kIP:       2.76092803E-01
    Old IP:        2.83646671E-01
    Old Delta IP: -7.55386890E-03
    Multipler:     2.00000000E+00
    New Omega:     3.71351644E+01

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.07499892617608  3.89486E-02  2.80117E-05 
      2  -200.07499890309481  2.30813E-08  6.29771E-06 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.79136808001297  3.89328E-02  1.94227E-05 
      2  -199.79136817063426 -9.06213E-08  3.58806E-06 DIIS

  *** Step Results ***

    Omega:     3.71351644E+01
    kIP:       2.76072570E-01
    IP:        2.83630732E-01
    Delta IP: -7.55816234E-03

  *** Step    9: Bracketing Step ***

    Old Omega:     3.71351644E+01
    Old kIP:       2.76072570E-01
    Old IP:        2.83630732E-01
    Old Delta IP: -7.55816234E-03
    Multipler:     2.00000000E+00
    New Omega:     7.42703288E+01

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.06206184514832  1.29371E-02  1.84099E-05 
      2  -200.06206423793171 -2.39278E-06  8.72199E-06 DIIS
      3  -200.06206432731184 -8.93801E-08  4.39471E-06 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.77844384931501  1.29243E-02  1.81647E-05 
      2  -199.77844313893050  7.10385E-07  4.33488E-06 DIIS
      3  -199.77844314585852 -6.92802E-09  1.31472E-06 DIIS

  *** Step Results ***

    Omega:     7.42703288E+01
    kIP:       2.76068962E-01
    IP:        2.83621181E-01
    Delta IP: -7.55221967E-03

  *** Step   10: Bracketing Step ***

    Old Omega:     7.42703288E+01
    Old kIP:       2.76068962E-01
    Old IP:        2.83621181E-01
    Old Delta IP: -7.55221967E-03
    Multipler:     2.00000000E+00
    New Omega:     1.48540658E+02

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.05859523685317  3.46909E-03  1.08523E-04 
      2  -200.05859487972691  3.57126E-07  4.22468E-05 DIIS
      3  -200.05859516113145 -2.81405E-07  9.46926E-06 DIIS
      4  -200.05859569846612 -5.37335E-07  1.55898E-06 DIIS
      5  -200.05859679838147 -1.09992E-06  1.37303E-06 DIIS
      6  -200.05859489011800  1.90826E-06  3.12264E-08 DIIS
      7  -200.05859599307624 -1.10296E-06  6.34785E-07 DIIS
      8  -200.05859561189189  3.81184E-07  2.36257E-07 DIIS
      9  -200.05859720692544 -1.59503E-06  2.25280E-06 DIIS
     10  -200.05859516206760  2.04486E-06  2.02907E-07 DIIS
     11  -200.05859598358271 -8.21515E-07  1.30529E-06 DIIS
     12  -200.05859608900772 -1.05425E-07  1.69919E-07 DIIS
     13  -200.05859701692478 -9.27917E-07  4.07924E-07 DIIS
     14  -200.05859558753383  1.42939E-06  6.94030E-07 DIIS
     15  -200.05859559624122 -8.70739E-09  3.89857E-07 DIIS

   => N-1 SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -199.77499862442195  3.44452E-03  7.11673E-05 
      2  -199.77498419401570  1.44304E-05  1.30840E-05 DIIS
      3  -199.77498508698778 -8.92972E-07  5.92445E-06 DIIS
      4  -199.77498461671905  4.70269E-07  2.85819E-06 DIIS
      5  -199.77498414694301  4.69776E-07  1.84086E-06 DIIS
      6  -199.77498460123715 -4.54294E-07  2.47224E-06 DIIS
      7  -199.77498579795039 -1.19671E-06  1.19734E-06 DIIS
      8  -199.77498418277162  1.61518E-06  1.22920E-06 DIIS
      9  -199.77498520235551 -1.01958E-06  2.12296E-06 DIIS
     10  -199.77498446849589  7.33860E-07  3.23192E-06 DIIS
     11  -199.77498500315130 -5.34655E-07  3.47384E-06 DIIS
     12  -199.77498499350349  9.64781E-09  2.15874E-06 DIIS

  *** Step Results ***

    Omega:     1.48540658E+02
    kIP:       2.76158294E-01
    IP:        2.83610603E-01
    Delta IP: -7.45230888E-03

  *** Step   11: Bracketing Step ***

    Old Omega:     1.48540658E+02
    Old kIP:       2.76158294E-01
    Old IP:        2.83610603E-01
    Old Delta IP: -7.45230888E-03
    Multipler:     2.00000000E+00
    New Omega:     2.97081315E+02

   => N SCF <=

   Iter               Energy      Delta E      Delta D Stat
      1  -200.05883010855257 -2.34512E-04  1.66549E-03 
      2  -200.05703877443074  1.79133E-03  5.99147E-04 DIIS
      3  -200.05757140345247 -5.32629E-04  1.56189E-04 DIIS
      4  -200.05761641650849 -4.50131E-05  7.44515E-05 DIIS
      5  -200.05773086932965 -1.14453E-04  3.80218E-05 DIIS
      6  -200.05772825805474  2.61127E-06  8.39170E-06 DIIS
      7  -200.05772082396783  7.43409E-06  3.37451E-06 DIIS
      8  -200.05771613154741  4.69242E-06  1.21714E-05 DIIS
      9  -200.05777023654679 -5.41050E-05  3.69340E-07 DIIS
     10  -200.05777168977792 -1.45323E-06  1.56022E-06 DIIS
     11  -200.05775992347466  1.17663E-05  6.37379E-05 DIIS
     12  -200.05749929527715  2.60628E-04  1.45981E-04 DIIS
     13  -200.05771999264852 -2.20697E-04  1.66918E-04 DIIS
     14  -200.05746709614127  2.52897E-04  5.43008E-04 DIIS
     15  -200.05789285994086 -4.25764E-04  4.97925E-04 DIIS
     16  -200.05743962126360  4.53239E-04  5.14045E-04 DIIS
     17  -200.05793961233198 -4.99991E-04  3.47960E-04 DIIS
     18  -200.05776519145095  1.74421E-04  1.29970E-04 DIIS
     19  -200.05759534394116  1.69848E-04  2.61779E-05 DIIS
     20  -200.05746076006128  1.34584E-04  3.80263E-05 DIIS
     21  -200.05766340052168 -2.02640E-04  1.02135E-04 DIIS
     22  -200.05738285693411  2.80544E-04  2.70456E-04 DIIS
     23  -200.05768088947352 -2.98033E-04  2.73192E-04 DIIS
     24  -200.05744339684290  2.37493E-04  9.64296E-05 DIIS
     25  -200.05750594471300 -6.25479E-05  4.18645E-05 DIIS
     26  -200.05762753462318 -1.21590E-04  3.43753E-05 DIIS
     27  -200.05771267665335 -8.51420E-05  2.60942E-04 DIIS
     28  -200.05740129439488  3.11382E-04  6.35650E-05 DIIS
     29  -200.05735467692594  4.66175E-05  1.50000E-04 DIIS
     30  -200.05779372195929 -4.39045E-04  6.16721E-05 DIIS
     31  -200.05744562823318  3.48094E-04  8.36126E-05 DIIS
     32  -200.05770217091958 -2.56543E-04  4.63766E-05 DIIS
     33  -200.05747052526181  2.31646E-04  6.02995E-05 DIIS
     34  -200.05732689027252  1.43635E-04  8.30387E-05 DIIS
     35  -200.05731128308810  1.56072E-05  1.61055E-04 DIIS
     36  -200.05777104821078 -4.59765E-04  1.32991E-04 DIIS
     37  -200.05738708241020  3.83966E-04  1.52204E-04 DIIS
     38  -200.05772483582101 -3.37753E-04  7.05495E-05 DIIS
     39  -200.05740546741291  3.19368E-04  1.58933E-04 DIIS
     40  -200.05770303422398 -2.97567E-04  1.55065E-04 DIIS
     41  -200.05787232985418 -1.69296E-04  1.37176E-04 DIIS
     42  -200.05767885603137  1.93474E-04  6.52188E-05 DIIS
     43  -200.05777033636161 -9.14803E-05  8.76720E-06 DIIS
     44  -200.05770953116996  6.08052E-05  2.48374E-05 DIIS
     45  -200.05773138848011 -2.18573E-05  1.95827E-05 DIIS
     46  -200.05772244424423  8.94424E-06  6.30215E-06 DIIS
     47  -200.05771392347464  8.52077E-06  7.29084E-06 DIIS
     48  -200.05771987273201 -5.94926E-06  1.16649E-06 DIIS
     49  -200.05772925338428 -9.38065E-06  1.27410E-05 DIIS
     50  -200.05773507536085 -5.82198E-06  1.05316E-05 DIIS
     51  -200.05773957720089 -4.50184E-06  2.19533E-05 DIIS
     52  -200.05772630838038  1.32688E-05  8.32990E-06 DIIS
     53  -200.05772890495970 -2.59658E-06  7.09447E-06 DIIS
     54  -200.05770911917895  1.97858E-05  1.22667E-05 DIIS
     55  -200.05772936903577 -2.02499E-05  6.29742E-06 DIIS
     56  -200.05771997162816  9.39741E-06  5.16588E-06 DIIS
     57  -200.05773197359301 -1.20020E-05  5.14997E-06 DIIS
     58  -200.05772979684986  2.17674E-06  9.73437E-07 DIIS
     59  -200.05768991641503  3.98804E-05  1.09894E-06 DIIS
     60  -200.05772741386247 -3.74974E-05  9.91003E-06 DIIS
     61  -200.05772677286865  6.40994E-07  1.39586E-05 DIIS
     62  -200.05774862448965 -2.18516E-05  4.01862E-06 DIIS
     63  -200.05770768258682  4.09419E-05  5.42484E-06 DIIS
     64  -200.05772183957490 -1.41570E-05  4.12407E-06 DIIS
     65  -200.05772000787297  1.83170E-06  1.01140E-06 DIIS
     66  -200.05772693679103 -6.92892E-06  2.32049E-06 DIIS
     67  -200.05770884915452  1.80876E-05  2.92048E-05 DIIS
     68  -200.05773972157729 -3.08724E-05  3.54574E-05 DIIS
     69  -200.05774238750527 -2.66593E-06  3.06655E-05 DIIS
     70  -200.05773533220690  7.05530E-06  2.51732E-05 DIIS
     71  -200.05772426695043  1.10653E-05  1.43109E-05 DIIS
     72  -200.05771998611101  4.28084E-06  3.20834E-05 DIIS
     73  -200.05775894587975 -3.89598E-05  2.60574E-05 DIIS
     74  -200.05771675639727  4.21895E-05  2.31441E-05 DIIS
     75  -200.05774969956983 -3.29432E-05  3.26601E-05 DIIS
     76  -200.05771227248709  3.74271E-05  1.84063E-05 DIIS
     77  -200.05771377029032 -1.49780E-06  2.38551E-05 DIIS
     78  -200.05774104309697 -2.72728E-05  3.24338E-05 DIIS
     79  -200.05771017734114  3.08658E-05  1.03560E-05 DIIS
     80  -200.05772291661938 -1.27393E-05  1.92812E-05 DIIS
     81  -200.05771133268763  1.15839E-05  3.66415E-05 DIIS
     82  -200.05772763621877 -1.63035E-05  1.56396E-05 DIIS
     83  -200.05773158689439 -3.95068E-06  1.07940E-05 DIIS
     84  -200.05772971841085  1.86848E-06  6.69704E-06 DIIS
     85  -200.05771546573359  1.42527E-05  9.28282E-06 DIIS
     86  -200.05772790283007 -1.24371E-05  1.56457E-05 DIIS
     87  -200.05772112081516  6.78201E-06  3.41128E-06 DIIS
     88  -200.05771643939786  4.68142E-06  2.41674E-05 DIIS
     89  -200.05775017481889 -3.37354E-05  3.22365E-05 DIIS
     90  -200.05771065316048  3.95217E-05  3.17694E-05 DIIS
     91  -200.05773775022254 -2.70971E-05  3.13223E-05 DIIS
     92  -200.05771225448024  2.54957E-05  2.86391E-05 DIIS
     93  -200.05774195330929 -2.96988E-05  1.94960E-05 DIIS
     94  -200.05770736947062  3.45838E-05  1.08208E-05 DIIS
     95  -200.05773979532486 -3.24259E-05  7.52661E-06 DIIS
     96  -200.05773371639432  6.07893E-06  3.14811E-06 DIIS
     97  -200.05771763223674  1.60842E-05  3.30973E-06 DIIS
     98  -200.05774966880546 -3.20366E-05  7.29746E-06 DIIS
     99  -200.05771713651006  3.25323E-05  6.83781E-06 DIIS
    100  -200.05771971796469 -2.58145E-06  1.09307E-05 DIIS
